src/libpriv/rpmostree-scripts.c

```diff
@@ -309,35 +303,16 @@ rpmostree_posttrans_run_sync (DnfPackage    *pkg,
 gboolean
 rpmostree_pre_run_sync (DnfPackage    *pkg,
                         Header         hdr,
-                        GHashTable    *ignore_scripts,
                         int            rootfs_fd,
                         GCancellable  *cancellable,
                         GError       **error)
 {
   for (guint i = 0; i < G_N_ELEMENTS (pre_scripts); i++)
     {
-      if (!run_known_rpm_script (&pre_scripts[i], pkg, hdr, ignore_scripts,
+      if (!run_known_rpm_script (&pre_scripts[i], pkg, hdr,
                                  rootfs_fd, cancellable, error))
         return FALSE;
     }
 
   return TRUE;
 }
-
-gboolean
-rpmostree_script_ignore_hash_from_strv (const char *const *strv,
-                                        GHashTable **out_hash,
-                                        GError **error)
```

Comment:
`parse_packages_strv()` is close but not quite the same.  But if something else has a version, seems simplest to delete this one right?